.github/workflows/release.yml

```diff
@@ -71,6 +71,8 @@ jobs:
         with:
           publish: yarn changeset publish
           version: yarn changeset:version-and-format
+          title: 'ci(changesets): version packages'
```

Comment:
This is the PR title, right? If so, I would keep the default.

Reply:
Yup. Wasn't sure what's used via a squash merge. The first commit or the title of the pr as the commit.